Druid-Tests/DRAbstractMockTest.class.st

```diff
@@ -6,42 +6,32 @@ Class {
 	#name : #DRAbstractMockTest,
```

Comment:
We should discuss this name later. An abstract mock test is testing mocks, and we do not test mocks in this project :)